public/views/partials/lib/flash/push.liquid

```diff
@@ -0,0 +1,18 @@
+{% liquid
+  if error_key
+    assign error = error_key | t
+  endif
+
+  if notice_key
+    assign notice = notice_key | t
+  endif
+
+  if info_key
+    assign info = info_key | t
+  endif
+  assign flash = null | hash_merge: error: error, notice: notice, info: info, from: context.location.pathname, now: force_clear
```

Comment:
`now: force_clear`

is it `now` or `force_clear`? you store it as now but you expect force_clear in pop.liquid

Reply:
also, maybe i'm wrong, but i think what you implemented here is a key-value store and not a stack (so it's not a pop-push logic, as it gets overwritten every time since it's not an array). With the current implementation you can push any messages, but you can only get the last one.

Reply:
`now` has been taken from internal rails implementation, which uses this name, that's why `flash` tag uses this keyword; `force_clear` however feels more understandable by developers who are not familiar with rails.  Not sure what's the best to do here :) 

Good point about